Makefile

```diff
@@ -73,6 +73,13 @@ clean-dirs:
 test:
 	./hack/cicd/run-tests.sh
 
+certs: clean-certs
+	@echo "Generating certificates"
+	./scripts/gen_cert.sh
```

Comment:
This should be required to be done manually by the user. As is being done in case of infrap4d.

Reply:
The script is the manual operation